lib/rdoc/ri/driver.rb

```diff
@@ -1152,13 +1152,36 @@ def interactive
   # Is +file+ in ENV['PATH']?
 
   def in_path? file
-    return true if file =~ %r%\A/% and File.exist? file
+    extensions = [''].union(RbConfig::CONFIG['EXECUTABLE_EXTS']&.split || [RbConfig::CONFIG['EXEEXT' || '']])
```

Comment:
```suggestion
    extensions = [RbConfig::CONFIG['EXEEXT']].union(RbConfig::CONFIG['EXECUTABLE_EXTS']&.split(" ") || [])
```

Reply:
Your suggestion offers `[".exe", ".com", ".cmd", ".bat"]` on windows but I expect `["", ".exe", ".com", ".cmd", ".bat"]`, works in case PAGER has filename with it's extension.

`['', RbConfig::CONFIG['EXEEXT']].union(RbConfig::CONFIG['EXECUTABLE_EXTS']&.split(" ") || [])` may work but I'm afraid that 'extension' is not a good name in this situation.
`additional_suffixes` or `candidate_suffixes` are my idea, should we change the name `extensions`?